lib/mage/Mage.js

```diff
@@ -222,6 +222,10 @@ Mage.prototype.getTask = function () {
 Mage.prototype.setupCoreLibs = function () {
 	var core = this.core;
 
+	// Load the configuration
+
+	core.config = require('../config');
```

Comment:
If the require call throws (because at the top-scope of `../config` something goes wrong), will that be logged? There is no logger on `mage.core` yet.